packages/loot-core/src/types/api-handlers.d.ts

```diff
@@ -155,4 +160,26 @@ export interface ApiHandlers {
   'api/rule-update': (arg: { rule: RuleEntity }) => Promise<RuleEntity>;
 
   'api/rule-delete': (arg: { id: string }) => Promise<boolean>;
+
+  'api/schedules-get': () => Promise<ScheduleEntity[]>;
+
+  'api/schedule-create': (arg: {
+    schedule: ScheduleEntity;
+    conditions: unknown[];
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind typing this? I know some of the other API methods don't yet have strict typing, but for all the new code added we're trying to enforce strict TS.